lib/common/client/middleware/response/snakecase.rb

```diff
@@ -0,0 +1,33 @@
+module Common
+  module Client
+    module Middleware
+      module Response
+        class Snakecase < Faraday::Response::Middleware
+
+          def on_complete(env)
+            if env.response_headers['content-type'] =~ /\bjson/
```

Comment:
I can't find the source that handles this, but you maybe be able to just add this to the connection setup like the example for FaradayMiddleware::ParseJson
`conn.response :snakecase, :content_type => /\bjson$/`
then you could remove this method as the next line is already what is defined in the base class
https://github.com/lostisland/faraday/blob/master/lib/faraday/response.rb#L16